ci/terraform/oidc/mfa-reset-jar-jwk.tf

```diff
@@ -0,0 +1,56 @@
+module "mfa_reset_jar_signing_jwk_role" {
+  source      = "../modules/lambda-role"
+  environment = var.environment
+  role_name   = "mfa-reset-jar-jwk-role"
+  vpc_arn     = local.authentication_vpc_arn
+
+  policies_to_attach = [
+    aws_iam_policy.mfa_reset_jar_kms_signing_jwk_policy.arn
+  ]
+}
+
+module "mfa_reset_jar_signing_jwk" {
+  source = "../modules/endpoint-module"
+
+  endpoint_name   = "reverification-jwk.json"
```

Comment:
why the choice of this endpoint name?

Reply:
Think I had a discussion with @ethanmills about this sometime last week, because I was unsure what to name it as the `/.well-known/mfa-reset-jwk.json` is currently used to serve the storage token signing JWKs

Reply:
I wonder if we should rename that one `/mfa-reset-token-jwk.json` and then use `/mfa-reset-jar-jwk.json` for this one?

Reply:
`storage-token-jwk.json` is probably a good bet, both because it matches orch and because it doesn't preclude issuing these tokens as part of a different interaction too.